python/setup.py

```diff
@@ -0,0 +1,33 @@
+import os
+import io
+
+from setuptools import setup, find_packages
+
+install_require = []
+with io.open('requirements.txt') as f:
+    install_require = [line.strip() for line in f if not line.startswith("#")]
+install_dev_require = []
+with io.open('requirements-dev.txt') as f:
+    install_dev_require = [line.strip() for line in f if not line.startswith("#")]
+
+setup(
+    name='statewize-python-sdk',
+    version="1.0.0",
+    author='Leonid Haimov',
+    author_email='leoisprogramming@gmail.com',
```

Comment:
As this is public info, we should put the company's details
```suggestion
    author='STATEWIZE',
    author_email='support@statewize.com',
```